tests/pipelines/fetchdatairidanext.nf.test

```diff
@@ -54,4 +54,58 @@ nextflow_pipeline {
             assert path("$launchDir/results/reads/SRR999934_2.fastq.gz").exists() == false
         }
     }
+
+    test("integration test with ENA") {
+
+        when {
+            params {
+                input = "$baseDir/tests/data/samplesheet.csv"
+                outdir = "test3_out"
```

Comment:
There isn't a problem with this, but why `test3_out` and not something like `output` or `results`?